test/Common.FunctionalTests/PublishedSitesFixture.cs

```diff
@@ -19,18 +19,19 @@ public class PublishedSitesCollection : ICollectionFixture<PublishedSitesFixture
 
     public class PublishedSitesFixture : IDisposable
     {
-        public TestApplication InProcessTestSite { get; } = new TestApplication(Helpers.GetInProcessTestSitesPath());
-        public TestApplication OutOfProcessTestSite { get; } = new TestApplication(Helpers.GetOutOfProcessTestSitesPath());
-        public TestApplication StartupExceptionWebsite { get; } = new TestApplication(Helpers.GetTestWebSitePath("StartupExceptionWebsite"));
-        public TestApplication OverriddenServerWebSite { get; } = new TestApplication(Helpers.GetTestWebSitePath("OverriddenServerWebSite"));
-        
+        public CachingApplicationPublisher InProcessTestSite { get; } = new CachingApplicationPublisher(Helpers.GetInProcessTestSitesPath());
```

Comment:
The field type can be the base ApplicationPublisher right?

Reply:
No, ApplicationPublisher doesn't have dispose.